enea_consts:
  i_pitch: 567               # [LSB] Initial arm pitch
  i_head_pitch: 663          # [LSB] Initial head pitch
  i_head_roll: 512            # [LSB] Initial head roll
  servo_speed: 200           # [LSB] Speed of the servomotor
  r_pitch:                   # [LSB] Range of arm pitch angles
    - 200
    - 750
  r_head_pitch:              # [LSB] Range of head pitch angles
    - 0
    - 800 
  r_head_roll:                # [LSB] Range of head roll angles
    - 0
    - 1023
  pitch_conv:  0.75          # [ ] Conversion of pitch increase to head pitch increase